src/index.ts

```diff
@@ -73,86 +73,100 @@ declare global {
        *
        * @defaultValue `process.cwd() + "/tasks"`
        */
       taskDirectory?: string;
       /**
-       * Override path to crontab file.
+       * The path to a file in which Graphile Worker should look for crontab schedules.
        *
        * @defaultValue `process.cwd() + "/crontab"`
        */
       crontabFile?: string;
       /**
-       * Number of jobs to run concurrently.
+       * Number of jobs to run concurrently on a single worker.
        *
        * @defaultValue `1`
        */
       concurrentJobs?: number;
-
       /**
-       * A list of file extensions (in priority order) that Graphile Worker
-       * should attempt to import directly when loading tasks. Defaults to
-       * `[".js", ".cjs", ".mjs"]`.
+       * A list of file extensions (in priority order) that Graphile Worker should
+       * attempt to import directly when loading task executors from the file system.
+       *
+       * @defaultValue `[".js", ".cjs", ".mjs"]`
        */
       fileExtensions?: string[];
-
       /**
        * How long in milliseconds after a gracefulShutdown is triggered should
-       * we wait to trigger the AbortController, which should cancel supported
-       * asynchronous actions?
+       * Graphile Worker wait to trigger the AbortController, which should cancel
+       * supported asynchronous actions?
        *
-       * @defaultValue `5000`
+       * @defaultValue `5_000`
        */
       gracefulShutdownAbortTimeout?: number;
-
       /**
-       * Set `true` to use the time as recorded by Node.js rather than
+       * Set to `true` to use the time as recorded by Node.js rather than
        * PostgreSQL. It's strongly recommended that you ensure the Node.js and
        * PostgreSQL times are synchronized, making this setting moot.
+       *
+       * @defaultValue `false`
        */
       useNodeTime?: boolean;
-
       /**
        * **Experimental**
        *
-       * How often should we scan for jobs that have been locked too long and
-       * release them? This is the minimum interval, we'll choose a time between
-       * this and `maxResetLockedInterval`.
+       * How often should Graphile Worker scan for and release jobs that have been
+       * locked too long? This is the minimum interval in milliseconds. Graphile
+       * Worker will choose a time between this and `maxResetLockedInterval`.
+       *
+       * @defaultValue `480_000`
        */
       minResetLockedInterval?: number;
       /**
        * **Experimental**
        *
-       * The upper bound of how long we'll wait between scans for jobs that have
-       * been locked too long. See `minResetLockedInterval`.
+       * In milliseconds, the upper bound of how long Graphile Worker will wait
+       * between scans for jobs that have been locked too long (see
+       * `minResetLockedInterval`).
+       *
+       * @defaultValue `600_000`
        */
       maxResetLockedInterval?: number;
-
```

Comment:
Please restore all of these blank newlines (and add any missing ones) - I use `{` and `}` in vim to navigate the code via blank lines; it also makes it easier to read the code - knowing that the comment applies to the entry below it rather than above it.